src/components/color_picker/color_picker.xml

```diff
@@ -21,13 +21,13 @@
           </div>
         </div>
         <div class="o-separator"/>
-        <div class="o-color-picker-section-name">
+        <div
+          class="o-color-picker-section-name o-color-picker-toggler"
+          t-on-click="toggleColorPicker">
           <span>Custom</span>
         </div>
-        <div class="colors-grid">
-          <div
-            class="o-color-picker-toggler o-color-picker-line-item"
-            t-on-click.stop="toggleColorPicker">
+        <div class="colors-grid o-color-picker-toggler" t-on-click.stop="toggleColorPicker">
```

Comment:
Actually, I'd remove the classes `.o-color-picker-toggler` completely.
In the tests, I would target directly elements with their existing classes `.o-color-picker-section-name` and `.colors-grid`. That would be more explicit that we want *those* elements to toggle the color picker.

What's no longer explicitly tested is that clicking on the "+" opens the color picker.
What we can do to test it more easily: add a custom class on the "+" (like `.o-color-picker-plus`) and target it

Reply:
the comment is still relevant :)

Reply:
ok does make sense, I didn't see the comments below Sorry about that